guardian/cdk
# chore(deps)!: Update aws-cdk libraries to 1.137.0

## What does this change?


Moving from 1.132.0 to 1.137 brings a few new features and bug fixes, most notably tag support added to `AWS::Logs::LogGroup` resources!

There are a few snapshot updates to reflect these new features and bug fixes. See the individual commits for more information.

We should update our Lambda constructs to create an explicit `LogGroup` to take advantage of tag support. See also https://github.com/guardian/cloudwatch-logs-management/issues/19.

BREAKING CHANGE: Update aws-cdk libraries to 1.137.0

## How to test



See CI pass.

Note: CI is now witnessing a lot of `console.log`s, this gets fixed in https://github.com/aws/aws-cdk/pull/18136.

## How can we measure success?


Keeping dependencies up to date (and unblocking https://github.com/guardian/cdk/pull/995).

## Have we considered potential risks?


n/a

## Checklist


[^1]: Consider whether this is something that will mean changes to projects that have already been migrated, or to the CDK CLI tool. If changes are required, consider adding a checklist here and/or linking to related PRs.
[^2]: If you are adding a new construct or pattern, has new documentation been added? If you are amending defaults or changing behaviour, are the existing docs still valid?
